app/src/androidTest/assets/features/publicPrivateKeyGeneration.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Public and private key generation
+  As a user
+  I want to have the app generate a public/private key pair when I use it for the first time
+  So that I don't have to generate a pair myself
```

Comment:
No they aren't, we are not concerned with encrypting the Bluetooth/NFC/whatever connection